arviz/tests/test_rcparams.py

```diff
@@ -135,6 +137,62 @@ def test_make_validate_choice(args, allow_none, typeof):
         assert value in accepted_values or value is None
 
 
+@pytest.mark.parametrize("allow_none", (True, False))
+@pytest.mark.parametrize("allow_auto", (True, False))
+@pytest.mark.parametrize(
+    "args", [(False, (1, 2)), (False, "auto"), (False, None), (False, "(1, 4)")]
```

Comment:
Very minor nit, no need to change it actually, there is no need to pass `raise_error` as it is always the same.

Reply:
Well, now that I'm aware of it, I feel like changing it. Can I? :)